apps/web/src/components/PoolStatusTable/PoolStatusTable.tsx

```diff
@@ -172,10 +179,31 @@ const PoolStatusTable = ({ user, pools }: UserInformationProps) => {
                           "Status message if the application is not suspended",
                       })}
                 </td>
+                <td
+                  data-h2-background-color="base(gray.light)"
+                  data-h2-padding="base(x.25)"
+                >
+                  <Link href={paths.poolCandidateApplication(candidate.id)}>
+                    {intl.formatMessage(
+                      {
+                        defaultMessage: "View {title} application",
+                        id: "+PCQ8p",
+                        description:
+                          "Link text to view a specific application of a candidate",
+                      },
+                      {
+                        title: intl.formatMessage(
+                          getPoolStream(candidate.pool.stream ?? ""),
```

Comment:
Just a small question for this 
This should never (probably) be null, but in this case an empty string in `getPoolStream()` will throw an error

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/f696e763-fbbd-460c-8c0a-8100f7b1086e)

Is that preferable to something like this to handle the null more gracefully? It'd sub in a empty space instead of erroring. 

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/4135b68f-7313-4c3c-8cd1-b02131b0814e)

Reply:
Good call, added in a N/A message if it isn't there.

[check for pool stream before rendering](https://github.com/GCTC-NTGC/gc-digital-talent/pull/8011/commits/3921cf95705fe0a83851186b0c52cef2360bad5b)